### ISSUE TITLE
attribute error

### ISSUE THREAD

**Issue by narges75z** [msminhas93/KerasImageDatagenTutorial#1]
Hi.

i am trying to get class indices and true labels and also related filenames for my test generator in keras. but it gives me attribute errors: " generator object has no attribute class_indices", "generator object has no attribute classes" and "generator object has no attribute filenames". However, these attributes are mentioned in the keras website document.I use keras 2.4.0 and ImageDataGenerator class and flow_from_dataframe or flow_from_directory in keras.I also tried keras 2.3 but did not work.I really appreciate if you help me with this problem.
